prow/plugins/label/label.go

```diff
@@ -228,7 +232,7 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	toRepeat := []string{}
 	if len(sigMatches) > 0 {
 		isMember, _ := gc.IsMember(ae.org, ae.login)
-		if !isMember {
+		if isMember {
```

Comment:
Good catch, I don't know exactly how this should work but certainly seems not doing the right thing. Fixed as instructed.